src/control/server/mgmt_svc.go

```diff
@@ -217,6 +219,10 @@ func (svc *mgmtSvc) GetAttachInfo(ctx context.Context, req *mgmtpb.GetAttachInfo
 		return nil, errors.Wrap(err, "unmarshal GetAttachInfo response")
 	}
 
+	resp.CRT_PHY_ADDR_STR = svc.cfg.CRT_PHY_ADDR_STR
+	resp.CRT_CTX_SHARE_ADDR = svc.cfg.CRT_CTX_SHARE_ADDR
+	resp.CRT_TIMEOUT = svc.cfg.CRT_TIMEOUT
+
```

Comment:
This should be guarded by a check to verify that svc.cfg is not nil, otherwise you could get a panic. Probably best to return with an error rather than omitting the agent config.

Reply:
Done